Orange/data/tests/test_variable.py

```diff
@@ -441,6 +443,8 @@ def test_make_proxy_disc(self):
         self.assertEqual(abc, abc1)
         self.assertEqual(abc, abc2)
         self.assertEqual(abc1, abc2)
+        self.assertEqual(hash(abc), hash(abc1))
+        self.assertEqual(hash(abc1), hash(abc2))
```

Comment:
Maybe add a test for non-equal hashes? I can imagine a scenario in which we mess the code so that all variables have the same hash. :)